beaker.gemspec

```diff
@@ -38,7 +38,7 @@ Gem::Specification.new do |s|
   ## aws-sdk-2 doesn't require json, so we can give this up when we move
 
   s.add_runtime_dependency 'hocon', '~> 1.0'
-  s.add_runtime_dependency 'net-ssh', '~> 3.2'
+  s.add_runtime_dependency 'net-ssh', '3.3.0beta1'
```

Comment:
Thanks - repushed.

Also didn't realize that GitHub was going to put the branch on this repo, rather than my fork - so we'll have to delete the branch once merged (assuming it works)